lib/pages/user_regattas.dart

```diff
@@ -13,9 +14,18 @@ class UserRegattasPage extends StatefulWidget {
 class _UserRegattasPageState extends State<UserRegattasPage> {
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(
+    return Scaffold(
       appBar: AppHeader(),
-      body: Text('Page with regattas that user is participating in')
+
+      // create body with button navigating to other page
```

Comment:
should these comments stay?

Reply:
This page is totally temp. I just has button so that we can navigate to the race page